CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Constellation Changes
 
+## Changes in October 2021
+
+-   Added `isRequired` in `PluginParameter` with a getter and a setter, which 
+    can be used to configure the required plugin parameters to mark as `*required`
+	in the swagger.
```

Comment:
How does the below sound? I felt API needed to be mentioned, and maybe that swagger was an interface.

Added `isRequired` in `PluginParameter` with a getter and a setter. This adds a `*required` annotation to the REST API parameter within the Swagger interface.

Reply:
How about this then, because the changes are only visible in the swagger currently:
"Added `isRequired` in `PluginParameter` with a getter and a setter, which can be used to configure the required plugin parameters to mark as `*required`	in the swagger interface of the REST API."?

There are no tests added in this area yet, and from what I can see, we can add tests to assert the endpoints, but it could be impossible or tricky to verify this specific change, and I don't see any value there anyway, as it's just enforcing the swagger users to enter the required parameters before executing.